UE4SS/src/LuaType/LuaUEnum.cpp

```diff
@@ -110,6 +110,240 @@ No overload found for function 'UEnum.ForEachName'.
+            std::string error_overload_not_found{R"(
+No overload found for function 'UEnum.GetEnumNameByIndex'.
+Overloads:
+#1: GetNameByValue(integer Value))"};
+
+            auto& lua_object = lua.get_userdata<UEnum>();
+
+            if (!lua.is_integer())
+            {
+                lua.throw_error(error_overload_not_found);
+            }
+
+            auto value = lua.get_integer();
+            auto enum_pair = lua_object.get_remote_cpp_object()->GetEnumNameByIndex(value);
+
+            LuaType::FName::construct(lua, enum_pair.Key);
+            lua.set_integer(enum_pair.Value);
+
+            return 2;
+        });
+
+        table.add_pair("InsertIntoNames", [](const LuaMadeSimple::Lua& lua) -> int {
+            // Function: InsertIntoNames
+            // Param #1: Enum key name
+            // Param #2: Enum value
+            // Param #3: Enum index
+            // (Optional) Param #4: Shift values
+            std::string error_overload_not_found{R"(
+No overload found for function 'UEnum.InsertIntoNames'.
+Overloads:
+#1: InsertIntoNames(string Name, integer Value, integer Index) // shiftValues = false
+#1: InsertIntoNames(string Name, integer Value, integer Index, bool shiftValues)"};
+
+            auto& lua_object = lua.get_userdata<UEnum>();
+
+            int32_t stack_size = lua.get_stack_size();
+
+            if (stack_size <= 0)
+            {
+                lua.throw_error("Function 'UEnum.InsertIntoNames' cannot be called with 0 parameters.");
+            }
+
+            StringType param_name{};
+            int param_value = 0;
+            int param_index = 0;
+            bool param_shift = false;
+
+            // P1 (Name), string
+            if (lua.is_string())
+            {
+                param_name = to_wstring(lua.get_string());
+            }
+            else
+            {
+                lua.throw_error("'UEnum.InsertIntoNames' could not load parameter for \"Name\"");
+            }
+
+            // P2 (Value), integer
+            if (lua.is_integer())
+            {
+                param_value = lua.get_integer();
+            }
+            else
+            {
+                lua.throw_error("'UEnum.InsertIntoNames' could not load parameter for \"Value\"");
+            }
+
+            // P3 (Index), integer
+            if (lua.is_integer())
+            {
+                param_index = lua.get_integer();
+            }
+            else
+            {
+                lua.throw_error("'UEnum.InsertIntoNames' could not load parameter for \"Index\"");
+            }
+
+            // (Optional) P4 (ShiftValues), boolean
+            if (lua.is_bool())
+            {
+                param_shift = lua.get_bool();
+            }
+
+            Unreal::FName key = Unreal::FName(param_name, Unreal::FNAME_Add);
+            auto pair = Unreal::TPair{key, Unreal::int64(param_index)};
+
+            lua_object.get_remote_cpp_object()->InsertIntoNames(pair, param_index, true);
```

Comment:
This `true` should be `param_shift` I believe

Reply:
No clue how I keep missing stuff like this...
Fixed.